packages/common/src/dto/applicant-create.dto.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  IsDateString,
+  IsNotEmpty,
+  IsOptional,
+  IsString,
+  Length,
+  IsObject,
+  IsBoolean,
+  IsInt,
+} from 'class-validator';
+
+export class ApplicantCreateDto {
+  @IsString()
+  @IsNotEmpty()
+  @Length(1, 256)
+  first_name!: string;
+
+  @IsString()
+  @Length(1, 256)
+  @IsOptional()
+  last_name?: string;
+
+  @IsString()
+  @IsNotEmpty()
+  @Length(1, 256)
+  profession!: string;
+
+  @IsString()
+  @Length(1, 256)
+  @IsOptional()
+  speciality?: string;
```

Comment:
I've noticed two different naming conventions for speciality (spreadsheet uses 'specialty'), I assume we should be using 'specialty' but we should decide 100% which we'll be moving forward with,

Reply:
Agree we will go with ‘specialty’ only. Let me update it.